kitsune/settings.py

```diff
@@ -1159,6 +1159,9 @@ def filter_exceptions(event, hint):
 ZENDESK_OS_FIELD_ID = config("ZENDESK_OS_FIELD_ID", default="360018604871", cast=int)
 ZENDESK_COUNTRY_FIELD_ID = config("ZENDESK_COUNTRY_FIELD_ID", default="360026463511", cast=int)
 
+# Products that allow un-authenticated users to submit support requests
+LOGINLESS_PRODUCTS = ["mozilla-account"]
```

Comment:
nit: I would go for something like `LOGIN_EXCEPTIONS`

Reply:
That makes sense.